examples/oit/oit.cpp

```diff
@@ -0,0 +1,683 @@
+/*
+* Vulkan Example - High dynamic range rendering
+*
+* Note: Requires the separate asset pack (see data/README.md)
+*
+* Copyright by Sascha Willems - www.saschawillems.de
+*
+* This code is licensed under the MIT license (MIT) (http://opensource.org/licenses/MIT)
+*/
```

Comment:
This needs to be updated, as this refers to the HDR sample. Please also add your copyright below mine, so people can see that this sample was contributed b ypu.

Reply:
Oh it's my mistake... I have changed typos and added copyright.